beesdoo_website_shift/views/res_config_views.xml

```diff
@@ -19,9 +19,15 @@
             <label for="irregular_shift_limit"/>
             <field name="irregular_shift_limit"/>
           </div>
+<!--          Leave this part in comment to hide it in control panel
+              Easier to roll back if the client change is mind  -->
+<!--          <div>-->
+<!--            <label for="highlight_rule"/>-->
+<!--            <field name="highlight_rule"/>-->
+<!--          </div>-->
```

Comment:
Just remove it. I understand that you will be faster by rolling back using comment. But for that we should use git, that one of its purpose, rolling back when needed. :)